src/locales/en-US/commands.json

```diff
@@ -110,231 +110,232 @@
+    "commandDescription": "Shows a random text from /r/copypasta."
   },
   "owo": {
-    "commandDescription": "Owoifies a sentence",
-    "commandUsage": "<sentence>",
-    "missingSentence": "You need to give me a sentence to owoify",
-    "tooLongTitle": "Your sentence is longer than 200 characters",
-    "tooLongDescription": "Try shortening your sentence down or separate it into multiple commands"
+    "commandDescription": "Owoifies your text.",
+    "commandUsage": "<text>",
+    "missingSentence": "You have to give me some text to owoify!",
+    "tooLongTitle": "Your text is longer than 200 characters!",
+    "tooLongDescription": "Try shortening your text or separate it into multiple commands!"
   },
   "daily": {
-    "commandDescription": "Claims your daily reward",
-    "alreadyClaimedTitle": "You've already claimed your daily reward.",
-    "alreadyClaimedDescription": "You can do it again in **{{time}}**",
-    "claimedSuccessfully": "You've received **$t(commons:currencyWithCount, { 'count': {{count}} })** as your daily reward"
+    "commandDescription": "Claims your daily reward.",
+    "alreadyClaimedTitle": "Already claimed",
+    "alreadyClaimedDescription": "You've already claimed your daily reward. You can do it again in **{{time}}**.",
+    "claimedSuccessfully": "You've received **$t(commons:currencyWithCount, { 'count': {{count}} })** as your daily reward!"
   },
   "money": {
-    "commandDescription": "Shows how many $t(commons:currency_plural) someone has",
+    "commandDescription": "Shows how many $t(commons:currency_plural) someone has.",
     "commandUsage": "[user to lookup]",
-    "youHave": "You have **$t(commons:currencyWithCount, { 'count': {{count}} })** in your account",
-    "someoneHas": "{{- user}} has **$t(commons:currencyWithCount, { 'count': {{count}} })** in their account"
+    "youHave": "You have **$t(commons:currencyWithCount, { 'count': {{count}} })** in your account.",
+    "someoneHas": "{{- user}} has **$t(commons:currencyWithCount, { 'count': {{count}} })** in their account."
   },
   "kiss": {
-    "commandDescription": "Kisses someone",
+    "commandDescription": "Kisses someone.",
     "commandUsage": "<user to kiss>",
-    "noMention": "You need to mention someone",
+    "noMention": "You have to mention someone.",
     "notYourself": "You can't kiss yourself!",
-    "success": "{{- kisser}} kissed {{- kissed}}"
+    "success": "{{- kisser}} kissed {{- kissed}}."
   },
   "handshake": {
-    "commandDescription": "Handshake someone",
+    "commandDescription": "Handshakes with someone.",
     "commandUsage": "<user to handshake>",
-    "noMention": "You need to mention someone",
-    "notYourself": "You can't handshake yourself!",
-    "success": "{{- handshaker}} handshaked {{- handshaked}}"
+    "noMention": "You have to mention someone!",
+    "notYourself": "You can't handshake with yourself!",
+    "success": "{{- handshaker}} handshaked with {{- handshaked}}"
```

Comment:
{{- handshaker}} shook hands with {{- handshaked}}

This is part of the comment above.